unbound-install.sh

```diff
@@ -1,5 +1,6 @@
 #!/bin/bash
 
+clear
```

Comment:
The script is pretty straightforward here (no complex menus, etc), so I don't think it's needed here